### ISSUE TITLE
Console.WriteLine has no overload with params formatting.

### ISSUE THREAD

**Issue by michaelcheers** [bridgedotnet/Bridge#1320]
### Expected

Compile successfully and output "1 2 3 4 5 6 7 8 9".

### Actual

C# Compilation Failed

```
Microsoft (R) Visual C# Compiler version 4.0.30319.34209

(31,3): error CS1501: No overload for method 'WriteLine' takes 10 arguments
\\fs6-n02\stor14wc1ord1\956235\live.bridge.net\web\content\Bridge\Builder\Bridge.dll: (Location of symbol related to previous error)
```

### Steps To Reproduce
[Bridge Fiddle](http://live.bridge.net/#4fa5887718907a03e9fde49c9228bfcc)
[Dot Net Fiddle](https://dotnetfiddle.net/j5zNem)
```csharp
public class App
{
    [Ready]
    public static void Main()
    {
		Console.WriteLine("{0} {1} {2} {3} {4} {5} {6} {7} {8}", 1, 2, 3, 4, 5, 6, 7, 8, 9);
    }
}
```
The solution is just to implement this:
```csharp
[Template("console.log(Bridge.String.format({format}, {args}))")]
public static extern void WriteLine(
	string format,
	params object[] args);
```

**UPD**: fixed along with #1304

**Comment by michaelcheers** [comment]
Pull Request: #1315 .

**Comment by geoffreymcgill** [comment]
Interesting how this overload is not noted in the [documentation](https://msdn.microsoft.com/en-us/library/system.console_methods%28v=vs.110%29.aspx), but it is in the source:

http://referencesource.microsoft.com/#mscorlib/system/console.cs,2138

I will add.

**Comment by michaelcheers** [comment]
@geoffreymcgill, It is noted on this [MSDN](https://msdn.microsoft.com/en-us/library/828t9b9h(v=vs.110).aspx) page.

**Comment by geoffreymcgill** [comment]
I added the missing `params`. Included the fix in the existing pull request. 

https://github.com/bridgedotnet/Bridge/pull/1319/commits/fd3bb5fae2e22a0d4fa70254fdcef727e5e641c1

**Comment by michaelcheers** [comment]
Me too, I added it to my [pull request](https://github.com/bridgedotnet/Bridge/pull/1315/commits/3874b9127823af4d7553f1982a7a56cf850c0dbe) (#1315).

**Comment by LeonidVeriga** [comment]
Fixed along with #1304
